build.sbt

```diff
@@ -52,7 +47,12 @@ libraryDependencies ++= {
     "com.typesafe.slick" %% "slick-hikaricp" % slickV,
     "mysql" % "mysql-connector-java" % "5.1.38",
     "org.liquibase" % "liquibase-core" % "3.3.5",
-    "ch.qos.logback" % "logback-classic" % "1.1.6"
+    "ch.qos.logback" % "logback-classic" % "1.1.6",
+    "com.typesafe.akka" %% "akka-testkit" % akkaV % "test",
+    "io.spray" %% "spray-testkit" % sprayV % "test",
+    "org.scalatest" %% "scalatest" % "2.2.4" % "test",
+    "org.mock-server" % "mockserver-netty" % "3.9.2" % "test",
+    "com.h2database" % "h2" % "1.4.191" % "test"
```

Comment:
Yes.  For this iteration, I don't want to make running Docker and MySQL mandatory for unit test.